pkg/operator/staticpod/certsyncpod/certsync_controller.go

```diff
@@ -79,6 +79,7 @@ func getSecretDir(targetDir, secretName string) string {
 func (c *CertSyncController) sync() error {
 	errors := []error{}
 
+	klog.Infof("Syncing configmaps: %v", c.configMaps)
```

Comment:
do we have a good printer on these? I doubt it.

Reply:
You'll get name + optional pairs from the default writer, which imo is sufficient for now.